test/support/test_no_sandbox_repo.ex

```diff
@@ -8,7 +8,7 @@ defmodule AshPostgres.TestNoSandboxRepo do
   end
 
   def installed_extensions do
-    ["ash-functions", "uuid-ossp", "pg_trgm", "citext"] --
+    ["ash-functions", "uuid-ossp", "pg_trgm", "citext", AshPostgres.TestCustomExtension.extension()] --
```

Comment:
This is cool :) I think we should just have them put the module here, though, and if they do we will call `.extension()` on it :)

Reply:
Hi @zachdaniel, good tip!
I've fixed that and the formatting / credo issues 🙂